manifests/server.pp

```diff
@@ -310,6 +310,7 @@
   $nodeid                           = $zabbix::params::server_nodeid,
   $listenport                       = $zabbix::params::server_listenport,
   $sourceip                         = $zabbix::params::server_sourceip,
+  String $logtype                   = $zabbix::params::server_logtype,
```

Comment:
do you know if only specific values are allowed here? In that case you could use the Enum. Otherwise prohibit empty strings with `String[1]`.